### PR TITLE
Fix: Fixed issue where Alt-Up focused on the column header

### LINKED ISSUE #10567: Bug: Alt-up focuses on the tab
### Description

When I press `Alt - arrow up`, files does not move up a folder, but instead focuses the tab header:

image

The tooltip of the move-up button indicates that it should work however:

image

### Steps To Reproduce

It reproduces always on any folder for me.

### Steps to fix

- Navigate correctly 

### Files Version

Version: 2.3.12.0 OS Version: 10.0.22621.819

### Windows Version

Windows 11 Business Version	22H2 OS build	22621.819 

### Relevant Assets/Logs

[debug.log](https://github.com/files-community/Files/files/10090459/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/10090460/debug_fulltrust.log)

### LINKED ISSUE #10926: Bug: Right click menu doesn't work on path bar
### Description

When you need to paste a path with the context menu in the path bar, the context menu flashes and disappears.

### Steps To Reproduce

1.Open Files.
2.Right click on the path bar.
3.The context menu will flash and disappear.

### Steps to fix

Fix the bug that the context menu flashes and disappears.

### Files Version

Version:2.4.8.0 OS Version:10.0.25267.1000

### Windows Version

Windows 11 22H2 25267.1000

### Relevant Assets/Logs

[debug.log](https://github.com/files-community/Files/files/10349310/debug.log)

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #10567 
- Closes #10926

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

**Comment by yaira2** [comment]
Can you share a screen recording of the new behavior?

**Comment by ferrariofilippo** [comment]
Here it is

https://user-images.githubusercontent.com/102259289/205747735-4e4ff7ba-dfff-4b70-b766-4f168a94b797.mov

**Comment by yaira2** [comment]
Is it selecting the "Name" column header now?

**Comment by QuaintMako** [comment]
> Is it selecting the "Name" column header now?

The `Name` header selection is probably a side effect of #10439. The detailed view focused first the header while the view is loading the items, and then the first element.

**Comment by yaira2** [comment]
What's the best path forward here?

**Comment by QuaintMako** [comment]
An idea I cannot test for now, we may get away with a proper unfocus on line 449 in `src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs`.

Feels like a makeshift solution though.

**Comment by yaira2** [comment]
The focus is set correctly when clicking the toolbar buttons, can we try to replicate this with the shortcuts?

**Comment by ferrariofilippo** [comment]
@yair100 @QuaintMako 
I fixed the focus problem. Basically, the issue was that `FileList` had `IsTabStop` set to `false` for some reason when it should default to `true`.
Can you tell me if now it's working as intended?

**Comment by ferrariofilippo** [comment]
@yair100 do you know how to fix the problem with the pipeline?

**Comment by yaira2** [comment]
Did you try running the tests locally to see why it's failing?

File: src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml [hunk 1]
```diff
@@ -219,13 +219,15 @@
 					tui:ScrollViewerExtensions.EnableMiddleClickScrolling="{x:Bind IsMiddleClickToScrollEnabled, Mode=OneWay}"
 					x:FieldModifier="public"
 					AllowDrop="{x:Bind InstanceViewModel.IsPageTypeSearchResults, Converter={StaticResource BoolNegationConverter}, Mode=OneWay}"
+					AutomationProperties.Name="{helpers:ResourceString Name=FilesAndFolders}"
```
Comment (yaira2) [inline src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml:222]:
```suggestion
					AutomationProperties.Name="{helpers:ResourceString Name=FilesAndFolders}"
					AutomationProperties.AccessibilityView="Raw"
```

File: src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml [hunk 2]
```diff
@@ -219,13 +219,16 @@
...
```
Comment (yaira2) [inline src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml:222]:
```suggestion
```

**Comment by yaira2** [comment]
The behavior is better but it still doesn't match the behavior when clicking the `up` button in the toolbar.

**Comment by ferrariofilippo** [comment]
> The behavior is better but it still doesn't match the behavior when clicking the `up` button in the toolbar.

Can you tell me more about this?

**Comment by yaira2** [comment]
Notice the black rectangle as the focus changes
https://user-images.githubusercontent.com/39923744/208308528-b104a686-9935-4b76-8b77-bbee21cd64c7.mp4

**Comment by ferrariofilippo** [comment]
With the latest commit the bug is almost fixed (occurs in < 1/10). I don't think it can be 100% fixed, since you can't focus on specific items, but you need to focus the `ListView`.

File: src/Files.App/Views/LayoutModes/GridViewBrowser.xaml.cs [hunk 3]
```diff
@@ -428,6 +423,29 @@ private async void FileList_PreviewKeyDown(object sender, KeyRoutedEventArgs e)
...
+					// Unfocus the ListView so keyboard shortcut can be handled (alt + shift + "+")
+					this.Focus(FocusState.Pointer);
+					return;
+				}
+
+				// Unfocus the GridView so keyboard shortcut can be handled
+				switch (e.Key)
+				{
+					case VirtualKey.Up:
+						NavToolbar?.Up.Focus(FocusState.Pointer);
```
Comment (gave92) [inline src/Files.App/Views/LayoutModes/GridViewBrowser.xaml.cs:439]: Any reason GridView is handled differently wrt to Details & Column? (`NavToolbar?.Up.Focus()` VS `this.Focus()`)

Comment (ferrariofilippo) [reply src/Files.App/Views/LayoutModes/GridViewBrowser.xaml.cs:439]: Using `this.Focus()` in Columns makes the behaviour of shortcuts unpredictable. Sometimes the focus remains on the `GridView` and this element handles arrows (resulting in focus moving between cells). I can't reproduce this issue in Details.

**Comment by gave92** [comment]
@yair100 could you test with the latest changes if focus behaviour is good?

**Review by yair100 — APPROVED** [review]
LGTM

**Pull request merged**